Cargo.toml

```diff
@@ -27,7 +27,7 @@ url = "1"
 # used in filesizeformat filter
 humansize = "1"
 # used in date format filter
-chrono = "0.2"
+chrono = ">=0.2, < 0.4"
```

Comment:
Ok, I was just allowing chrono 0.2 to be used with tera, but I'm ok with 0.3 only ^^